models/CouponModel.php

```diff
@@ -301,8 +302,14 @@ public function create_coupon( array $data ) {
 	 * @return mixed true|false on insert, void if not insert-able
 	 */
 	public function insert_applies_to( string $applies_to, array $applies_to_ids, $coupon_code ) {
-		$specific_applies = array( self::APPLIES_TO_SPECIFIC_BUNDLES, self::APPLIES_TO_SPECIFIC_COURSES, self::APPLIES_TO_SPECIFIC_CATEGORY );
-		if ( in_array( $applies_to, $specific_applies ) ) {
+		$specific_applies = array(
+			self::APPLIES_TO_SPECIFIC_BUNDLES,
+			self::APPLIES_TO_SPECIFIC_COURSES,
+			self::APPLIES_TO_SPECIFIC_CATEGORY,
+			self::APPLIES_TO_SPECIFIC_MEMBERSHIP_PLANS,
```

Comment:
`self::APPLIES_TO_SPECIFIC_MEMBERSHIP_PLANS` This should come from the Subscription addon using hook